mods/cnc/maps/nod04a/nod04a.lua

```diff
@@ -0,0 +1,320 @@
+Atk3CellTriggerActivator = { CPos.New(18,18), CPos.New(17,18), CPos.New(16,18), CPos.New(15,18), CPos.New(14,18), CPos.New(13,18), CPos.New(12,18), CPos.New(11,18), CPos.New(24,17), CPos.New(23,17), CPos.New(22,17), CPos.New(21,17), CPos.New(20,17), CPos.New(19,17), CPos.New(17,17), CPos.New(16,17), CPos.New(15,17), CPos.New(14,17), CPos.New(13,17), CPos.New(12,17), CPos.New(11,17) }
+Atk4CellTriggerActivator = { CPos.New(29,28), CPos.New(28,28), CPos.New(29,27), CPos.New(28,27), CPos.New(29,26), CPos.New(28,26), CPos.New(29,25), CPos.New(28,25), CPos.New(29,24), CPos.New(28,24), CPos.New(29,23), CPos.New(28,23), CPos.New(29,22), CPos.New(28,22) }
+GcivCellTriggerActivator = { CPos.New(51,17), CPos.New(50,17), CPos.New(49,17), CPos.New(48,17), CPos.New(47,17), CPos.New(46,17), CPos.New(45,17), CPos.New(44,17), CPos.New(43,17), CPos.New(42,17), CPos.New(41,17), CPos.New(40,17), CPos.New(39,17), CPos.New(38,17), CPos.New(37,17), CPos.New(36,17), CPos.New(35,17), CPos.New(52,16), CPos.New(51,16), CPos.New(50,16), CPos.New(49,16), CPos.New(48,16), CPos.New(47,16), CPos.New(46,16), CPos.New(45,16), CPos.New(44,16), CPos.New(43,16), CPos.New(42,16), CPos.New(41,16), CPos.New(40,16), CPos.New(39,16), CPos.New(38,16), CPos.New(37,16), CPos.New(36,16), CPos.New(35,16) }
+DelxCellTriggerActivator = { CPos.New(42,20), CPos.New(41,20), CPos.New(40,20), CPos.New(39,20), CPos.New(38,20) }
+DelyCellTriggerActivator = { CPos.New(31,28), CPos.New(30,28), CPos.New(31,27), CPos.New(30,27), CPos.New(31,26), CPos.New(30,26), CPos.New(31,25), CPos.New(30,25), CPos.New(31,24), CPos.New(30,24) }
+DelzCellTriggerActivator = { CPos.New(18,20), CPos.New(17,20), CPos.New(16,20), CPos.New(15,20), CPos.New(14,20), CPos.New(13,20), CPos.New(12,20), CPos.New(11,20), CPos.New(25,19), CPos.New(24,19), CPos.New(23,19), CPos.New(22,19), CPos.New(21,19), CPos.New(20,19), CPos.New(19,19), CPos.New(18,19), CPos.New(17,19), CPos.New(16,19), CPos.New(15,19), CPos.New(14,19), CPos.New(13,19), CPos.New(12,19), CPos.New(11,19), CPos.New(25,18), CPos.New(24,18), CPos.New(23,18), CPos.New(22,18), CPos.New(21,18), CPos.New(20,18), CPos.New(19,18) }
+ShowCivilianCellTriggerActivator = { CPos.New(43,29), CPos.New(43,30), CPos.New(43,31), CPos.New(43,32), CPos.New(43,33), CPos.New(43,34), CPos.New(43,35), CPos.New(43,36), CPos.New(43,37), CPos.New(43,38), CPos.New(43,39) }
+
+Atk3TriggerCounter = 2
+
+Atk1TriggerFunctionTime = DateTime.Seconds(20)
+XxxxTriggerFunctionTime = DateTime.Seconds(50)
+YyyyTriggerFunctionTime = DateTime.Minutes(1) + DateTime.Seconds(40)
+ZzzzTriggerFunctionTime = DateTime.Minutes(2) + DateTime.Seconds(30)
+
+NodCivilians = { { { 'c9' }, { spawnciv0.Location } }, { { 'c8' }, { spawnciv1.Location } }, { { 'c7' }, { spawnciv2.Location } }, { { 'c4' }, { spawnciv3.Location } }, { { 'c3' }, { spawnciv4.Location } }, { { 'c1' }, { spawnciv5.Location } }, { { 'c4' }, { spawnciv6.Location } }, { { 'c5' }, { spawnciv7.Location } }, { { 'c2' }, { spawnciv8.Location } } }
```

Comment:
These just popping up at those locations looks a bit weird. Is it possible to find another solution? Maybe have them placed in the map from the start and only change the owner when the trigger activates? Or let them come out of their houses or something.